### PR TITLE
Add day

### PR DESCRIPTION
**Pull request by drowsy810301**
add day in remaining time of index's contest table

### CODE REVIEW

File: index/static/js/contestTimer.js [hunk 1]
```diff
@@ -68,11 +68,21 @@ function getRestTime() {
...
+            var h = parseInt(result % 24);
+            result /= 24;
+            var d = parseInt(result);
             m = checkTime(m);
             s = checkTime(s);
             h = checkTime(h);
-            remainings[i].innerHTML = h + ":" + m + ":" + s;
+            var day;
+            if(d > 1)
+                day = d + " days , ";
```
Comment (nickelcheng) [inline index/static/js/contestTimer.js]: ```days,``` (no space after s)

File: index/static/js/contestTimer.js [hunk 2]
```diff
@@ -68,11 +68,21 @@ function getRestTime() {
...
+            else if(d == 1)
+                day = d + " day , ";
```
Comment (nickelcheng) [inline index/static/js/contestTimer.js]: ```day,``` (no space after s)

**Comment by nickelcheng** [comment]
test ok, +1 besides the comment

**Comment by nickelcheng** [comment]
BTW, can you modify your contest count down as well?
If not, maybe @Changron will need to do that.

**Comment by drowsy810301** [comment]
comment fixed
okay,count down issue will be in the next pr

File: index/static/js/contestTimer.js [hunk 3]
```diff
@@ -68,11 +68,21 @@ function getRestTime() {
...
             result /= 60;
-            var h = parseInt(result);
+            var h = parseInt(result % 24);
+            result /= 24;
+            var d = parseInt(result);
             m = checkTime(m);
             s = checkTime(s);
             h = checkTime(h);
-            remainings[i].innerHTML = h + ":" + m + ":" + s;
+            var day;
```
Comment (bruce3557) [inline index/static/js/contestTimer.js]: replace with ```var day = ""``` and remove else part

Comment (bruce3557) [inline index/static/js/contestTimer.js]: Just curious. Any function about ```checktDay(d)```?

**Comment by bruce3557** [comment]
Seems good to me beside comments

File: index/static/js/contestTimer.js (hunk 3 above)
Comment (drowsy810301) [inline index/static/js/contestTimer.js]: the function about checkTime is to add a '0' if the number < 10,and I think 01 day or 02 days would be weird.

**Comment by drowsy810301** [comment]
comment fixed

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**